ros/lib/utils.py

```diff
@@ -158,6 +158,11 @@ def calculate_percentage(numerator, denominator):
         return 0
 
 
+def systems_with_profile(org_id):
```

Comment:
Will it make sense to include one more function argument which will accept the extra filters specific to context?

Reply:
I've only seen filter for `number_of_recommendations`. Not generic. Is it still needed?

Reply:
> I've only seen filter for `number_of_recommendations`. Not generic. Is it still needed?

Then, it won't be needed. One query - do we really required `number_of_recommendations` filter? In which scenario system will have a performance profile record without any recommendation?

Reply:
`Optimized` state archive.